### PR TITLE
some packaging files 

### PR DESCRIPTION
**Pull request by necrose99**
some packaging files 

rpm spec skell 

debian skell , ie python3-maryam prepackaged.. /stdeb auto.. 

MSI file if you care to add to releases... 

gitignore add more ignore directories for  **py* setup.py bdist_wheel** or etc to not get unwanted leftovers.. 
ie bdist_msi bdist_rpm etc... 

testing my gentoo ebuild for @pentoo 

pipefile you can regen for devsecops/vultest code...

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[requires]
+python_version = "3.8"
+deps =
+    lxml
+    flask
+    requests
+    bs4
+    cloudscraper
+commands =
+    python setup.py test
```
Comment (saeeddhqan) [inline Pipfile]: @necrose99 currently, we don't have test files.

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
```
Comment (saeeddhqan) [inline tox.ini:1]: currently, we don't have test files.

File: Pipfile (hunk 1 above)
Comment (necrose99) [reply Pipfile]: centos 8 / debian 9 live in vmware , vmw-fs to folder ...  

your welcome to add to a branch as templates , for now.. 

my next task is to test ebuild for Gentoo/@pentoo 

pull to pentoo

**Comment by saeeddhqan** [comment]
gitignore is good.
why should we have Debian files and tox? just because of pentoo?

**Comment by necrose99** [comment]
https://salsa.debian.org/necrose99/maryam/  **chown to do (ie change owner )** 
 , anyways can rig docker to continually deb this / mirror repo.. and done  for debian  ie apt install python3-maryam 
rpm spec can likewise be offered unto that community... 

hopefully lures in more devs....

 i switched  this tox basicly mirrors **make test** ...  it will report on compat  with newer pythons automatically hopefully. 
can pull to another branch and burn off unwanteds.. 
tox makes @gentoo devs possibly qa happy ... if pentoo package moves over to gentoo someday.. 

as for pentoo...  ive added a pull.

**Comment by saeeddhqan** [comment]
@necrose99 Do you have an email address? or slack?

**Comment by saeeddhqan** [comment]
If you need any change, you can do it. you can have access to the repository.

**Pull request closed without merging**